demos/python_demos/object_detection_demo_yolov3_async/object_detection_demo_yolov3_async.py

```diff
@@ -176,16 +185,20 @@ def parse_yolo_region(blob, resized_image_shape, original_im_shape, params, thre
             except OverflowError:
                 continue
             # Depends on topology we need to normalize sizes by feature maps (up to YOLOv3) or by input shape (YOLOv3)
-            input_width = w_exp * params.anchors[2 * n] / (resized_image_w if params.isYoloV3 else params.side)
-            input_height = h_exp * params.anchors[2 * n + 1] / (resized_image_h if params.isYoloV3 else params.side)
+            w = w_exp * params.anchors[2 * n] / size_normalizer[0]
+            h = h_exp * params.anchors[2 * n + 1] / size_normalizer[1]
```

Comment:
Can you rename single char vars. It's hard to search for their usages with Ctrl+f. Here and in `def scale_bbox(x, y, h, w, class_id, confidence, im_h, im_w, is_proportional):`